nx.json

```diff
@@ -7,7 +7,9 @@
           "build:bundle",
           "build:transpile",
           "build:types",
-          "lint:eslint"
+          "lint:eslint",
+          "test",
```

Comment:
Let's maybe only start with caching `test:unit`, as the integration tests (for example) also make sure stuff keeps working with new versions of e.g. browsers, which is not really deterministic in this way.

WDYT @AbhiPrasad  / @Lms24  / @lforst ?

Reply:
yes that makes sense to me

Reply:
Reverted, I'll leave this one to you to revisit :)